diff --git a/dba/create-database.sh b/dba/create-database.sh
index b1dd1fa..b7d0c7a 100755
--- a/dba/create-database.sh
+++ b/dba/create-database.sh
@@ -10,22 +10,29 @@ DBADIR="`dirname \"$0\"`"
 
 INSTALL_NOTE_FN="`mktemp -t tmp.XXXXXXXXXX`"
 
-testawldir() {
-  [ -f "${1}/dba/awl-tables.sql" ]
-}
 
-#
-# Attempt to locate the AWL directory
-AWLDIR="${DBADIR}/../../awl"
-if ! testawldir "${AWLDIR}"; then
-  AWLDIR="/usr/share/awl"
-  if ! testawldir "${AWLDIR}"; then
-    AWLDIR="/usr/local/share/awl"
-    if ! testawldir "${AWLDIR}"; then
-      echo "Unable to find AWL libraries"
-      exit 1
+# Candidate locations of the AWL directory
+awldirs="${DBADIR}/../../awl
+/usr/share/awl
+/usr/share/php/awl
+/usr/local/share/awl"
+
+# Disable globbing and use newline as seperator
+set -f; IFS='
+'
+for d in $awldirs ; do
+    if [ -f "${d}/dba/awl-tables.sql" ] ; then
+	AWLDIR="${d}/dba/awl-tables.sql"
+	break
     fi
-  fi
+done
+
+# Renable file globbing and reset seperator 
+set +f; unset IFS
+
+if [ -z "${AWLDIR}" ] ; then
+    echo "Unable to find AWL libraries"
+    exit 1
 fi
 
 export AWL_DBAUSER=davical_dba
