--- cmake/FindPostgreSQL.cmake.orig	2016-10-09 16:11:58.995601785 -0400
+++ cmake/FindPostgreSQL.cmake	2016-10-09 16:17:05.527242865 -0400
@@ -22,6 +22,9 @@
     set(POSTGRESQL_FOUND TRUE)
 else(POSTGRESQL_INCLUDE_DIR AND POSTGRESQL_LIBRARIES AND POSTGRESQL_EXECUTABLE)
 
+  if(NOT "$ENV{PG_CONFIG}" STREQUAL "")
+    set(POSTGRESQL_PG_CONFIG $ENV{PG_CONFIG})
+  else(NOT "$ENV{PG_CONFIG}" STREQUAL "")
     if(NOT "${POSTGRESQL_BIN}" STREQUAL "")
         # Checking POSTGRESQL_PG_CONFIG 
         find_program(POSTGRESQL_PG_CONFIG NAMES pg_config
@@ -36,6 +39,7 @@
             /usr/lib/postgresql/*/bin/
             )
     endif(NOT "${POSTGRESQL_BIN}" STREQUAL "")
+  endif(NOT "$ENV{PG_CONFIG}" STEQUAL "")
 
     message(STATUS "POSTGRESQL_PG_CONFIG is " ${POSTGRESQL_PG_CONFIG})
 
