diff -ur pgrouting-2.6.0.orig/doc/conf.py.in pgrouting-2.6.0/doc/conf.py.in
--- pgrouting-2.6.0.orig/doc/conf.py.in	2018-03-20 11:49:37.000000000 -0400
+++ pgrouting-2.6.0/doc/conf.py.in	2018-09-14 06:48:48.181278427 -0400
@@ -240,7 +240,7 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    ('index_man', 'pgrouting', u'pgRouting Reference', u'pgRouting Contributors', 7)
+    ('index', 'pgrouting', u'pgRouting Reference', u'pgRouting Contributors', 7)
 ]
 
 # If true, show URL addresses after external links.
