diff -Naruw geos-3.4.0.orig/include/geos/platform.h.in geos-3.4.0/include/geos/platform.h.in
--- geos-3.4.0.orig/include/geos/platform.h.in	2013-08-11 04:23:47.000000000 +0000
+++ geos-3.4.0/include/geos/platform.h.in	2013-08-11 11:51:52.337156511 +0000
@@ -96,8 +96,9 @@
    // It does leave a version in std.
 #  define ISNAN(x) (std::isnan(x))
 # elif defined(__sun) || defined(__sun__)
+#  define _XOPEN_SOURCE 600 // force iso/math_c99
 #  include <math.h>
-#  define ISNAN(x) (::isnan(x))
+#  define ISNAN(x) (std::isnan(x))
 # endif
 #endif
 
